frontend/src/metabase/components/Link.tsx

```diff
@@ -1,22 +1,27 @@
-import React from "react";
 import cx from "classnames";
-import PropTypes from "prop-types";
-import { Link as ReactRouterLink } from "react-router";
-import styled from "styled-components";
-import { display, color, hover, space } from "styled-system";
-
-import { stripLayoutProps } from "metabase/lib/utils";
 import Tooltip from "metabase/components/Tooltip";
+import { stripLayoutProps } from "metabase/lib/utils";
+import React, { ReactNode } from "react";
+import { Link as ReactRouterLink, LinkProps } from "react-router";
+import styled from "styled-components";
+import { color, display, hover, space } from "styled-system";
```

Comment:
We usually keep all third-party imports at the top (see the [JS style guide note](https://github.com/metabase/metabase/blob/master/docs/developers-guide/frontend.md#javascript-conventions))

Reply:
Thanks, fixed!